src/Core/Commander/Providers/WMS_Provider.js

```diff
@@ -109,32 +106,21 @@ WMS_Provider.prototype.getColorTexture = function(tile, layer, bbox, pitch) {
     if (result.texture !== undefined) {
         return Promise.resolve(result);
     }
-    return this.ioDriverImage.read(url).then(function(image) {
-
-        var texture = this.cache.getRessource(image.src);
 
-        if(texture)
-            result.texture = texture;
-        else
-        {
-            result.texture = new THREE.Texture(image);
-            result.texture.needsUpdate = true;
-            result.texture.generateMipmaps = false;
-            result.texture.magFilter = THREE.LinearFilter;
-            result.texture.minFilter = THREE.LinearFilter;
-            result.texture.anisotropy = 16;
-            result.texture.url = url;
```

Comment:
So yeah, I couldn't see any use of this parameter in the code, but I may have missed something. Don't hesitate to point potential problems to me.